scene/3d/mesh_instance_3d.cpp

```diff
@@ -457,15 +457,23 @@ void MeshInstance3D::create_debug_tangents() {
 		MeshInstance3D *mi = memnew(MeshInstance3D);
 		mi->set_mesh(am);
 		mi->set_name("DebugTangents");
-		add_child(mi, true);
-#ifdef TOOLS_ENABLED
+		return mi;
+	}
 
-		if (is_inside_tree() && this == get_tree()->get_edited_scene_root()) {
-			mi->set_owner(this);
-		} else {
-			mi->set_owner(get_owner());
-		}
-#endif
+	return nullptr;
+}
+
+void MeshInstance3D::create_debug_tangents() {
+	MeshInstance3D *mi = create_debug_tangents_node();
+	if (!mi) {
+		return;
+	}
+
+	add_child(mi, true);
+	if (is_inside_tree() && this == get_tree()->get_edited_scene_root()) {
```

Comment:
This was previously wrapped in `#ifdef TOOLS_ENABLED`, but I think it's fine to leave it, because `get_edited_scene_root()` just returns null in non-editor builds.